tests/test_api.py

```diff
@@ -56,12 +56,12 @@ def setup_teardown(
         yield
 
         # Reset rate limits
-        state.delete_config("global_concurrent_limit")
-        state.delete_config("global_per_second_limit")
-        state.delete_config("project_concurrent_limit")
-        state.delete_config("project_concurrent_limit_1")
-        state.delete_config("project_per_second_limit")
-        state.delete_config("date_align_seconds")
+        state.delete_config_value("global_concurrent_limit")
```

Comment:
why did this name need to change?

Reply:
`{set/get/delete}_config()` could be slightly ambiguous in terms of the return value since other "get" functions return more than just value, eg `get_current_configs()`

Reply:
but this is a `snuba.state` function, why did it have to change?

Reply:
Whooops bad grep